src/models/company.dao.ts

```diff
@@ -0,0 +1,101 @@
+import dataSource from "./database";
+import { whereBuilder } from "./builder/queryBuilder";
+
+const createCompany = async (companyName: string) => {
+    await dataSource.query(
+        `
+        INSERT INTO
+          companies
+          (company_name)
+        VALUES
+          (?)
+        `,
+        [companyName]
+    );
+};
+
+const getCompanyId = async (companyName: string) => {
+    const companyId = await dataSource.query(
+        `
+        SELECT
+          id
+        FROM
+          companies
+        WHERE
+          company_name = ?
+        `,
+        [companyName]
+    );
+    return companyId;
+};
+
+const createCompanyMainMember = async (companyId: string, userId: string) => {
+    await dataSource.query(
+        `
+        INSERT INTO
+          company_members
+          (companies_id, users_id, is_main_member)
+        VALUES
+          (?, ?, ?)
+        `,
+        [companyId, userId, 1]
+    );
+};
+
+const createCompanyMember = async (companyId: string, userId: string) => {
+    await dataSource.query(
+        `
+      INSERT INTO
+        company_members
+        (companies_id, users_id, is_main_member)
+      VALUES
+        (?, ?, ?)
+      `,
+        [companyId, userId, 0]
+    );
+};
+
+const createCompanyResidences = async (companyId: string, startDate: string, endDate: string) => {
+    await dataSource.query(
+        `
+        INSERT INTO
+          company_residences
+          (companies_id, start_date, end_date)
+        VALUES
+          (?, ?, ?)
+        `,
+        [companyId, startDate, endDate]
+    );
+};
+
+const getCompanies = async (searchOption: CompanySerarchOption) => {
```

Comment:
아래에서 100으로 확정하기 보다는, 다른 값으로 변경할 수 있도록 인자로 설정하고, 기본값을 100으로 부여해주세요 
```suggestion
const getCompanies = async (searchOption: CompanySerarchOption, limit?: number = 100) => {
```

Reply:
반영했습니다. 감사합니다~!